blog/tag/tips.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
@lrowang In addition to this file, you'll need to add the tips tag to `_data/tags.yml` 🏷